src/components/landing/CommunityListingHeader.js

```diff
@@ -0,0 +1,98 @@
+import CustomizableAvatar from "../common/customavatar/CustomizableAvatar";
+import Link from "next/link";
+import Image from "next/image";
+
+const CommunityHeader = ({ listing }) => {
+  const photo = "/community dashoard  img 1.jpg";
+  const Manager_Avatar = "/avarar 1.png";
+  const AVATAR_WIDTH = 50;
+  const AVATAR_HEIGHT = 50;
+  const SmallAvatar_WIDTH = 20;
+  const SmallAvatar_HEIGHT = 20;
+  const LOGO_WIDTH = 20;
+  const LOGO_HEIGHT = 20;
+  return (
+    <header className="">
+      <div className=" bg-primary-lite p-5 md:flex lg:flex items-center justify-between px-10">
+        <article className="flex items-center pb-4">
+          <CustomizableAvatar
+            src={photo}
+            width={AVATAR_WIDTH}
+            height={AVATAR_HEIGHT}
+            avatarImage={"rounded-full"}
+            avatarCustomClass={"mx-2"}
+          />
+          <h1 className="text-grey-dark text-xl font-medium capitalize">
+            {listing?.community_name}
+          </h1>
+        </article>
+
+        <div className="flex items-center px-4">
+          <div className="flex items-center bg-white rounded-lg py-2 px-4 text-justify">
+            <CustomizableAvatar
```

Comment:
Now for this use of `<CustomizableAvatar>` this is fine, because this static avatar image communicates that this element displays the name of a person. So this is fine.